### PR TITLE
docs: Use new Conan 1.53 features

### PR DESCRIPTION
**Pull request by prince-chrismc**
since it's now available #13753 🚀

### CODE REVIEW

File: docs/package_templates/autotools_package/all/conanfile.py [hunk 1]
```diff
@@ -58,20 +58,10 @@ def config_options(self):
...
-        try:
-            del self.settings.compiler.libcxx
-        except Exception:
-            pass
-        try:
-            del self.settings.compiler.cppstd
-        except Exception:
-            pass
+        self.options.rm_safe("libcxx")
+        self.options.rm_safe("cppstd")
```
Comment (SpaceIm) [inline docs/package_templates/autotools_package/all/conanfile.py:64]:
```suggestion
        self.settings.rm_safe("compiler.libcxx")
        self.settings.rm_safe("compiler.cppstd")
```

File: docs/package_templates/cmake_package/all/conanfile.py [hunk 2]
```diff
@@ -60,20 +60,10 @@ def config_options(self):
...
-        try:
-            del self.settings.compiler.libcxx
-        except Exception:
-            pass
-        try:
-            del self.settings.compiler.cppstd
-        except Exception:
-            pass
+        self.options.rm_safe("libcxx")
+        self.options.rm_safe("cppstd")
```
Comment (SpaceIm) [inline docs/package_templates/cmake_package/all/conanfile.py:66]:
```suggestion
        self.settings.rm_safe("compiler.libcxx")
        self.settings.rm_safe("compiler.cppstd")
```

File: docs/package_templates/meson_package/all/conanfile.py [hunk 3]
```diff
@@ -63,20 +63,10 @@ def config_options(self):
...
-        try:
-            del self.settings.compiler.libcxx
-        except Exception:
-            pass
-        try:
-            del self.settings.compiler.cppstd
-        except Exception:
-            pass
+        self.options.rm_safe("libcxx")
+        self.options.rm_safe("cppstd")
```
Comment (SpaceIm) [inline docs/package_templates/meson_package/all/conanfile.py:69]:
```suggestion
        self.settings.rm_safe("compiler.libcxx")
        self.settings.rm_safe("compiler.cppstd")
```

File: docs/package_templates/msbuild_package/all/conanfile.py [hunk 4]
```diff
@@ -40,20 +40,10 @@ def config_options(self):
...
-        try:
-            del self.settings.compiler.libcxx
-        except Exception:
-            pass
-        try:
-            del self.settings.compiler.cppstd
-        except Exception:
-            pass
+        self.options.rm_safe("libcxx")
+        self.options.rm_safe("cppstd")
```
Comment (SpaceIm) [inline docs/package_templates/msbuild_package/all/conanfile.py:46]:
```suggestion
        self.settings.rm_safe("compiler.libcxx")
        self.settings.rm_safe("compiler.cppstd")
```

**Review by SpaceIm — CHANGES_REQUESTED** [review]

**Comment by prince-chrismc** [comment]
Good catch! Dangerous copy past strikes again

**Review by jwillikers — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Comment by SpaceIm** [comment]
So 1.53.0 is deployed?

**Comment by prince-chrismc** [comment]
It was with the other config pr!

**Pull request merged**